.env

```diff
@@ -30,7 +30,7 @@ LLVM=10
 CLANG_TOOLS=8
 RUST=nightly-2020-04-22
 GO=1.12
-NODE=11
+NODE=14
```

Comment:
Does this PR change the minimum supported node version?

Reply:
No we still work on node 11, but node changed the behavior of the `stream.readable` property, which we're checking in [`streams-node-tests.ts`](https://github.com/apache/arrow/blob/63264ce2ef37e55b1561927ebb28ba9804271cfa/js/test/unit/ipc/reader/streams-node-tests.ts#L212-L213). We used to check for true, but now it's false.